--- extconf.rb.o        2007-10-09 15:08:42.000000000 +0200
+++ extconf.rb  2007-10-09 15:07:54.000000000 +0200
@@ -35,8 +35,8 @@
   raise "ncurses header file not found"
 end

-if have_library("ncurses", "wmove")
-  curses_lib = "ncurses"
+if have_library("ncursesw", "wmove")
+  curses_lib = "ncursesw"
 elsif have_library("pdcurses", "wmove")
   curses_lib = "pdcurses"
 else
@@ -120,11 +120,11 @@

 puts "checking for the panel library..."
 if have_header("panel.h")
-  have_library("panel", "panel_hidden")
+  have_library("panelw", "panel_hidden")
 end
 puts "checking for the form library..."
 if have_header("form.h")
-  have_library("form", "new_form")
+  have_library("formw", "new_form")
 end

 create_makefile('ncurses')
